_ratelimits.md

```diff
@@ -18,7 +18,7 @@ The following table indicates the current rate limits:
 | POST /users                                | 10 / 10-min window         | N/A                          |
 
 <aside class="notice">
```

Comment:
I think we can remove this section since it links back to this documentation. /cc @ruimarinho